src/Donors/ServiceProvider.php

```diff
@@ -8,8 +8,10 @@
 use Give\Donors\CustomFields\Controllers\DonorDetailsController;
 use Give\Donors\Exceptions\FailedDonorUserCreationException;
 use Give\Donors\ListTable\DonorsListTable;
+use Give\Donors\Migrations\addPhoneColumn;
```

Comment:
```suggestion
use Give\Donors\Migrations\AddPhoneColumn;
```

Reply:
Resolved in 078a7d30bf7b851381f77f1b4785a2a5e0315de7